packages/core/src/registries/ElementRegistry.ts

```diff
@@ -17,7 +18,11 @@ const register = new Map<string, ElementRegistrationItem>();
 
 const upgrade = (name: string, definition: ElementConstructor): void => {
   definition.applyThemeStyles(CustomStyleRegistry.get(name));
-  customElements.define(name, definition);
+  const isElementDefined = customElements.get(name);
+  // For localhost, define custom element even when it's been defined already to inform developers about the issue.
+  if (isLocalhost || !isElementDefined) {
```

Comment:
This should improve test coverage.
```suggestion
  if (isLocalhost ||
      /* istanbul ignore next */ !isElementDefined) {
```

Reply:
confirmed that it doesn't improve test coverage. Marking as resolved.